phpunit-integration-legacy-elasticsearch.xml

```diff
@@ -13,6 +13,7 @@
         <env name="backendVersion" value="5" />
         <env name="fixtureDir" value="Elasticsearch" />
         <env name="elasticsearchServer" value="http://localhost:9200/" />
+        <env name="skipForNotImplementedException" value="" />
```

Comment:
could this maybe have a value to make it clear it means it will be skipped? _Like `true` or `1`._